src/common_constants.py

```diff
@@ -5,10 +5,11 @@
 @dataclass(frozen=True)
 class CommonConstants:
     GRAVITATIONAL_FORCE_EARTH: float = 9.81  # m/s^2
-    TIME_STEP: float = 0.01  # seconds
+    ROCKET_MASS = 1e5  # kg
```

Comment:
What do you think about naming variables in a pattern like "ROCKET_MASS_KG" or "TIME_STEP_SECONDS"?

Reply:
As we agreed earlier, I would prefer sticking to SI units by default and only adress units in variable names when it is ambiguous. This is the case for `pos`. Positions are described in either pixel coordinates or meters.